meta-facebook/yv35-cl/src/ipmi/plat_ipmi.c

```diff
@@ -54,15 +54,15 @@ bool add_sel_evt_record(addsel_msg_t *sel_msg) {
   msg->data[7] = (Self_I2C_ADDRESS << 1);              // generator id
   msg->data[8] = 0x00;                                // generator id
   msg->data[9] = evt_msg_version;                     // event message format version
-  msg->data[10] = 0x00;                                // sensor type, TBD
-  msg->data[11] = sel_msg->snr_name;                   // sensor name
-  msg->data[12] = sel_msg->evt_type;                   // sensor type
+  msg->data[10] = sel_msg->snr_type;                   // sensor type
+  msg->data[11] = sel_msg->snr_number;                 // sensor number
+  msg->data[12] = sel_msg->evt_type;                   // event type
   msg->data[13] = sel_msg->evt_data1;                  // sensor data 1
   msg->data[14] = sel_msg->evt_data2;                  // sensor data 2
   msg->data[15] = sel_msg->evt_data3;                  // sensor data 3
   record_id++;
 
-  status = ipmb_read(&msg, IPMB_inf_index_map[msg.InF_target]);
+  status = ipmb_read(&msg, IPMB_inf_index_map[msg->InF_target]);
   if (status == ipmb_error_failure) {
```

Comment:
Should msg be freed here?

Reply:
Yes, PRs basing on parent 100f4ba should all fix add_sel_evt_record as commit below so won't build fail.
https://github.com/facebook/OpenBIC/commit/186033d1b6a2f30b120ea941fb8802cb5359d704#diff-c14b4f8142a044f62bd5693e9d0f5581a627b85e74078ddd48901c72a53e30ebR66